### PR TITLE
feat: Add full example fixture

### LINKED ISSUE #19: Add example fixture
## Description

A fixture in golang rust bun or bash should be showing an example usage of golang. 

## Tasks

- [x] Add in the example toml file
- [x] Add ts scripts using mocktimism with op-viem
- [x] Add ts setup to contributing docs
- [x] Add way of running fixtures to contribute docs

### PR DESCRIPTION
**Pull request by roninjin10**
Add the example fixture for running mocktimism

Example fixture is a simple bun app. It has a test that mints an nft from l1 via a deposit tx and asserts the mint is successful on l2 using op-viem. Before the tests runs it spins up a mocktimism devnet.

Closes https://github.com/ethereum-optimism/mocktimism/issues/19

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* main
  * **PR #67** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/67" target="_blank">Graphite</a> 
    * **PR #68** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/68" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/mocktimism/68?utm_source=stack-comment).

File: example/index.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+			account
+		}).extend(walletL2OpStackActions)
+	}
+}
+
+export const mintOnL2 = async () => {
+	const tokenId = BigInt(420420)
+
+	console.info(`minting tokeId ${tokenId.toString()} on l2...`)
+	// TODO update version of op viem to mint on l2 with writeContractDeposit
```
Comment (roninjin10) [inline example/index.ts]: Will update this in a future pr after it publishes

File: example/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          "contracts": [
+            {
+              "name": "ExampleContract",
+              "addresses": {
+                "1": "0xFBA3912Ca04dd458c843e2EE08967fC04f3579c2",
+                "420": "0x1df10ec981ac5871240be4a94f250dd238b77901"
+              }
+            }
+          ]
+        }
```
Comment (roninjin10) [inline example/tsconfig.json]:
```suggestion
        "name": "@evmts/ts-plugin"
```
Unneeded config

File: example/bunfig.toml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Load plugin including EVMts plugin
+# EVMts is configured in the tsconfig.json
+# in future it will be configured via evmts.config.ts
```
Comment (roninjin10) [inline example/bunfig.toml]:
```suggestion
```

File: example/plugins.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { evmtsBunPlugin } from '@evmts/bun-plugin'
+import { plugin } from 'bun'
+
+// Load EVMts plugin to enable solidity imports
+// EVMts is configured in the tsconfig.json
+// in future it will be configured via evmts.config.ts
+plugin(evmtsBunPlugin())
```
Comment (roninjin10) [inline example/plugins.ts]:
```suggestion
plugin(evmtsBunPlugin())
```

**Review by hamdiallam — APPROVED** [review]

**Pull request merged**